ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -960,24 +937,22 @@ static bool TrySendRequestAndWaitForResponse(
     int* kauthResult,
     int* kauthError)
 {
+    // To be useful, the message needs to either provide an FSID/inode pair or a path
+    assert(vnodePath != nullptr || (vnodeFsidInode.fsid.val[0] != 0 || vnodeFsidInode.fsid.val[1] != 0));
```

Comment:
Is there anything about `vnodeFsidInode.inode` that we can/should assert on?

Reply:
I suspect we won't want to send messages for inode 0, but I'm not really willing to bet on it. In any case, this is intended as a basic check for programming errors when calling this function, nothing fancy.